### PR TITLE
Add bass

### PR DESCRIPTION
**Pull request by ghost**
Adds bass to Endless War

You fish the bass from the sea and then you can hit people with it

Like this:
http://giphygifs.s3.amazonaws.com/media/Tm4I6WTmu9cKk/giphy.gif

### CODE REVIEW

**Comment by minervatbd** [comment]
siiiick

File: ewcfg.py [hunk 1]
```diff
@@ -4263,6 +4308,14 @@ def wef_tool(ctn = None):
...
 		slime = fish_slime_saltwater
 	),
+        EwFish(
+		id_fish = "dirtybass",
+		str_name = "Dirty Bass",
+		rarity = fish_rarity_rare,
+		catch_time = None,
+		catch_weather = None,
+		str_desc = "A Dirty Bass Guitar. Maybe you should try cleaning it?",
+	),
```
Comment (magicksRampage) [inline ewcfg.py]: sounds like it should be an item rather than a fish

File: ewcfg.py [hunk 2]
```diff
@@ -2045,6 +2064,32 @@ def wef_tool(ctn = None):
...
+		id_weapon = "bass",
+		alias = [
+			"bass",
+			"BasS",
+			"basssss",
+			"baaassss",
+			"bassss",
+			"basssS"
+		],
+		str_crit = "**Critical hit!!** Through skilled swipes {name_player} manages to sharpy strike {name_target}’s {hitzone}.",
```
Comment (magicksRampage) [inline ewcfg.py]: sharpy typo

File: ewcfg.py [hunk 3]
```diff
@@ -2045,6 +2064,32 @@ def wef_tool(ctn = None):
...
+		str_miss = "**MISS!!** {name_player} swings and misses like a dumbass!",
+		str_equip = "You equip the bass guitar, a highly distorted and reverbed riff of unknown origin plays as you place the strap over your neck",
```
Comment (magicksRampage) [inline ewcfg.py]: add a period at the end

File: ewcfg.py [hunk 4]
```diff
@@ -2045,6 +2064,32 @@ def wef_tool(ctn = None):
...
+		str_weapon = "A bass guitar",
```
Comment (magicksRampage) [inline ewcfg.py]: lowercase a

File: ewcfg.py [hunk 5]
```diff
@@ -2045,6 +2064,32 @@ def wef_tool(ctn = None):
 		fn_effect = wef_tool,
 		str_description = "It's a super fishing rod",
 		acquisition = acquisition_smelting
+	),
+        EwWeapon(  # 13
+		id_weapon = "bass",
+		alias = [
+			"bass",
+			"BasS",
```
Comment (magicksRampage) [inline ewcfg.py]: aliases should always be all lowercase

**Review by magicksRampage — CHANGES_REQUESTED** [review]

File: ewcfg.py (hunk 1 above)
Comment (ghost) [reply ewcfg.py]: Bass comes from the sea.

Unless theirs away to get items from fishing but I didn't see anything like that.

File: ewcfg.py [hunk 6]
```diff
@@ -1771,6 +1771,25 @@ def wef_tool(ctn = None):
 		ctn.crit = True
 		ctn.slimes_damage *= 2
 
+# weapon effect function for "bass"
+def wef_bass(ctn = None):
```
Comment (magicksRampage) [inline ewcfg.py:1775]: the weapon effect is fine in principle, but meaty's weapon update is just around the corner, for which we changed all of the weapon effects. please talk to meaty about adjusting this effect for his update.

**Review by magicksRampage — CHANGES_REQUESTED** [review]

File: ewcfg.py (hunk 6 above)
Comment (ghost) [reply ewcfg.py:1775]: It is just the bat weapon effect

**Pull request merged**